sass/_main.scss

```diff
@@ -216,16 +216,17 @@ body.essay {
         display: block;
     }
     div.footnote-definition {
-      &[id="0"] {
+      &[id="1"] {
         margin-top: 4rem;
-        padding-top: 2rem;
+        padding-top: 4rem;
         border-top: 1px solid black;
       }
```

Comment:
That produces this, which I can't think is intentional:

![image](https://user-images.githubusercontent.com/20846414/90082847-da355480-dcde-11ea-85d6-84ff9b7358ce.png)